ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConfValidator.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.server;
+
+import com.netease.arctic.api.config.ConfigOption;
+import com.netease.arctic.api.config.Configurations;
+import com.netease.arctic.server.dashboard.utils.AmsUtil;
+
+import java.net.InetAddress;
+
+public class ArcticManagementConfValidator {
+  public static void validateConfig(Configurations configurations) {
+    // SERVER_EXPOSE_HOST config
+    if ("".equals(configurations.getString(ArcticManagementConf.SERVER_EXPOSE_HOST))) {
+      throw new IllegalArgumentException(
+          "configuration " + ArcticManagementConf.SERVER_EXPOSE_HOST.key() + " must be set");
+    }
+    InetAddress inetAddress =
+        AmsUtil.lookForBindHost(configurations.getString(ArcticManagementConf.SERVER_EXPOSE_HOST));
+    configurations.setString(ArcticManagementConf.SERVER_EXPOSE_HOST, inetAddress.getHostAddress());
+
+    // mysql config
+    if (ArcticManagementConf.DB_TYPE_MYSQL.equalsIgnoreCase(
```

Comment:
Now postgres is also supported I guess we can add.

Reply:
Username and password nullability check were added in 85ba2ef.